### ISSUE TITLE
No such action ‘spacemacs//helm-find-files-edit’ for this source

### ISSUE THREAD

**Issue by duianto** [syl20bnr/spacemacs#13038]
#### Description :octocat:
Trying to edit multiple files in a dired buffer from `helm-find-files` doesn't work.

#### Reproduction guide :beetle:
- `SPC f f` calls `lazy-helm/spacemacs/helm-find-files`
- `C-c C-e` calls `spacemacs/helm-find-files-edit`

*Observed behaviour:* :eyes: :broken_heart:
>apply: No such action ‘spacemacs//helm-find-files-edit’ for this source

*Expected behaviour:* :heart: :smile:
Open a dired buffer in editable mode.

#### Notes :notebook:
A similar issue was fixed for:
`SPC p p` calls `helm-projectile-switch-project`
`C-s` calls `spacemacs/helm-projectile-grep`

In this PR:
[helm] Fix searching in projects with C-s from SPC p p #13028

But the same solution of adding the function that's called as an action.
Inside the `(with-eval-after-load 'helm-files` section:
```elisp
      (with-eval-after-load 'helm-files
        (define-key helm-find-files-map
          (kbd "C-c C-e") 'spacemacs/helm-find-files-edit)
        ;; TESTING
        (helm-add-action-to-source
         "Edit `C-c C-e'" 'spacemacs//helm-find-files-edit
         helm-source-find-files))
```
source:
https://github.com/syl20bnr/spacemacs/blob/ccd6c48d462eef17fc0375c45b9f935e69c460e8/layers/%2Bcompletion/helm/packages.el#L120

doesn't work, instead it seems to break all helm buffers after restarting Spacemacs.
>Error (use-package): helm/:config: No buffer named *helm*

It might be some kind of loading order issue, because it works to add the action by evaluating it manually after Spacemacs has started.

#### System Info :computer:
- OS: windows-nt
- Emacs: 26.3
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. ccd6c48d4)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(autohotkey auto-completion command-log debug emacs-lisp git helm html javascript markdown multiple-cursors
            (org :variables org-agenda-files
                 '(("c:/Users/username/org/notes.org")))
            php python shell spell-checking syntax-checking themes-megapack treemacs typescript version-control vue)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG SOUND NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS THREADS LCMS2

**Comment by duianto** [comment]
The issues seems to be that `helm-source-find-files` is `nil` when trying to add the action in `helm/init-helm`.
In the `:init` or `:config` sections. Or in: `(with-eval-after-load 'helm-files`.
Possibly because Spacemacs lazy-loads helm.

The pending PR #13099, fixed it by adding the action in the hook: `helm-find-files-before-init-hook`

**Comment by smile13241324** [comment]
I have merged #13099, and I am not longer able to reproduce the issue. Could you also have a look and close the issue when its working also for you @duianto.
